src/coreclr/vm/loongarch64/stubs.cpp

```diff
@@ -1267,6 +1267,34 @@ void StubLinkerCPU::Init()
     //new (gLoadFromLabelIF) LoadFromLabelInstructionFormat();
 }
 
+static bool InRegister(UINT16 ofs)
+{
+    _ASSERTE(ofs != ShuffleEntry::SENTINEL);
+    return (ofs & ShuffleEntry::REGMASK);
+}
+
+static bool IsRegisterFloating(UINT16 ofs)
+{
+    _ASSERTE(InRegister(ofs));
+    return (ofs & ShuffleEntry::FPREGMASK);
+}
+
+static int GetRegister(UINT16 ofs)
+{
+    _ASSERTE(InRegister(ofs));
+    if (ofs & ShuffleEntry::FPREGMASK)
```

Comment:
Nit, it will never be called with FPREGMASK, you can turn it to an assertion.

Reply:
Thanks.